modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/odata/entity/v1_0/ObjectEntryEntityModel.java

```diff
@@ -175,8 +177,7 @@ private List<EntityField> _getRelatedObjectDefinitionEntityFields(
 			ObjectDefinition objectDefinition)
 		throws Exception {
 
-		_relatedObjectRelationships.add(
-			objectRelationship.getObjectRelationshipId());
+		_objectDefinitionsHandled.add(objectDefinition.getObjectDefinitionId());
 
 		ObjectDefinition relatedObjectDefinition = _getRelatedObjectDefinition(
```

Comment:
Maybe we should check if the relatedObjectDefinition is active?

Reply:
Mmmm, should we do that here?? I mean, it is true that we should check if the object definition is active or not before doing the filtering. 
What I am not sure about is if we should do this here, building the filtering, or when trying to retrieve the information.

Maybe we should discuss it internally